catalyst_voices/packages/catalyst_voices_models/analysis_options.yaml

```diff
@@ -3,4 +3,5 @@ include: package:catalyst_analysis/analysis_options.yaml
 analyzer:
   exclude: [build/**, lib/*.g.dart, lib/generated/**]
   errors:
+    implementation_imports: ignore
```

Comment:
Please ignore a single line where the problem happens, don't disable this lint rule for the whole package. You should be able to do something like this:

```
// ignore: implementation_imports
import 'package:bip39/src/wordlists/english.dart';
```